Gayathri_Ganesh_Tries_1.cpp

```diff
@@ -0,0 +1,82 @@
+// Time Complexity : O(N) N- number of alphabets in given word
+// Space Complexity :O(N)
+// Did this code successfully run on Leetcode : No
+// Any problem you faced while coding this : Yes
+
+
+// Your code here along with comments explaining your approach
+//Code in C++
+
+
+class Trie {
+    class TrieNode {
+        public:
+        bool isEnd;
+        TrieNode *children[26]; //26 as only lowercase letters
+        TrieNode() {        //constructor to initilaize children[] and isEnd members
+            for(int i=0;i<26;i++)
+            {
+                this->children[i]=NULL;
+            }
+            this->isEnd=false;
+        } //empty constructor
```

Comment:
This part is redundant we don't need it. Like in class we have an empty constructor in trienode